### ISSUE TITLE
Line attachments on power poles/towers

### ISSUE THREAD

**Issue by flacombe** [streetcomplete/StreetComplete#4825]
### General
Affected tag(s) to be modified/added: [line_attachment](https://wiki.openstreetmap.org/wiki/Key:line_attachment)
Question asked: **How power line is attached on this support?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [X] 🚧 To be added tag is established and has a useful purpose
- [X] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [X] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
- [X] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [X] 🕓 Applies to a reasonable number of map data (Worth the effort)

### Ideas for implementation

**Element selection:** Attachments may be complex and sometimes a single support may have different attachments leading to complex tagging notation like line_attachment=(suspension|pin|suspension)|(anchor). That's not what we expect for streetcomplete quest.
We should only select supports with:
* no line_attachment=* key.
* only one line passing by power supports

**Metadata needed:** This quest is worldwide and no local metadata applies.

**Proposed UI:**
Drawings are took from wiki and could be enhanced/redrawn to meet streetcomplete's standards. We could use photos as well.
![streetcomplete_attachment](https://user-images.githubusercontent.com/5690599/219903806-6ec58c16-992a-4c18-a0f7-cfe2d0194f34.png)

**Comment by matkoniecz** [comment]
>  To be added tag is established and has a useful purpose

Is there some, even theoretical use of this data except "can be used for 3D rendering"?

**Comment by flacombe** [comment]
Yes, there are several use cases that aren't currently documented on wiki.
At least in France, where power lines are public property, we had investigated for years now how to get the most suitable database to help operation around power poles.
It covers:
* Computer aided design for additional roll out on the same support (optical fibre mostly)
* Environmental risks study and scoring vulnerability of power lines versus surrounding forests, buildings... even meteorological conditions
* Daily operation and asset management for operators and contractors (who contribute back to osm as well)

It helps saving dozen of gallon of gasoline each day and help decarbonize power network operation.

OSM is the best option to gather such knowledge the operarors had lost along years or are unable to manage on their own.

**Comment by westnordost** [comment]
The illustrations are nice, however I think that this will be still too complex for the average-joe user and thus lead to a non-negligible amount of incorrect data added. E.g. in particular, it is not absolutely clear what the green things are and that in the anchor-picture, they are supposed to point at the viewer (just like the black dots in the other picture are pointing at the viewer).

Also, see @matkoniecz . Professional users ... I think do not want to trust crowdsourced data by non-experts that is collected in a somewhat non-comprehensive manner (i.e. with STreetComplete)

**Comment by flacombe** [comment]
Hi @westnordost 
Illustration was only a suggestion, not what should finally be used in a quest.
We could also use pictures for more clear situations.
We find some in wiki: https://wiki.openstreetmap.org/wiki/Tag%3Aline_attachment%3Danchor

> Professional users ... I think do not want to trust crowdsourced data

Why ?
Professional users already trust OSM actually: https://www.openstreetmap.org/user/InfosReseaux/diary/396044

**Comment by westnordost** [comment]
My point is, that they shouldn't trust OSM data on line attachments on power towers when it is recorded by laymen. As I wrote, the current pictures are not clear enough and I do not think that it is possible to find pictures (that are displayed at that size) to be clear enough.

**Comment by flacombe** [comment]
> they shouldn't trust OSM data on line attachments on power towers when it is recorded by laymen.

Current experiments comparing very local test datasets from power operators and osm data show very good success rate and we are confident in building the French national database for power poles from crowdsourced data.

> I do not think that it is possible to find pictures

When I look to "wood" picture of power poles material quest, I find it clear enough for "suspension" option we are currently discussing.
I can find similar pictures for other options.

**Comment by westnordost** [comment]
> Current experiments comparing very local test datasets from power operators and osm data show very good success rate and we are confident in building the French national database for power poles from crowdsourced data.

With laymen I meant StreetComplete users. I expect that OpenStreetMap contributors who currently add this data are much more interested (and more informed) about this thing than StreetComplete users will ever be.

**Comment by flacombe** [comment]
That's not what we currently observe in contributors' profile, both about OSM contributors and streetcomplete users on existing quests.
500 contributors got involved in power pole project in 2021 and most of them aren't power users.

Actually it's painful to not have user friendly tool to encourage such contribution. I don't worry about further contributors goodwill to gather the data.

**Comment by Jean-BaptisteC** [comment]
@westnordost I understand that you think main Stc users are not interested or doesn't have topic knowledge. Maybe add the quest but by default she is disabled? I think drawings need to more big for better comprehension.

**Comment by mnalis** [comment]
@Jean-BaptisteC As an alternative if it turns out not to be good fit for StreetComplete; perhaps [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (_"Expert Edition"_) fork of StreetComplete might be a better fit for such quest, if you'd like easiness of StreetCopmplete on-the-ground mapping, but some domain knowledge is needed. 

SCEE is generally less concerned with all quests being easy to answer for newbie mappers (as all extra quests are disabled by default, and it is expected that target clientele are more advanced users which will only enable stuff they are well versed and interested in), so perhaps you may want to suggest it [there](https://github.com/Helium314/SCEE/issues) if you think it would fit your use case.

(of course if it turns out to be acceptable here in upstream StreetComplete itself afterall, all the better!)

**Comment by Jean-BaptisteC** [comment]
@westnordost what are the conditions for you take a decision about this quest?
This quest can be very useful especially in places with severe bad weather like recently in France with the Cyaran storm because power lines are more or less resistant to wind following material poles/towers or how power lines are attached to the poles.
To give an example following a big storm in 1999, the main electrical network operator in France changed how lines are attached to towers to improve storm resistance.
We can take a similar example with a forest fire, if a region has a big forest fire, the electrical operator can replace wood poles with steel poles near of forest to improve fire resistance.
And these data can improve how decisions are taken to improve the resilience of power networks.

**Comment by qugebert** [comment]
tbh, i think the operators already have way more detailed information about their network design and won't use openstreetmap as a reliable data source (at least for aspects, where they have their own, more detailed documents) for operational decisions. 

Nevertheless, here a cross-reference (maybe interesting/helpful): There is a new preset PR for this here https://github.com/openstreetmap/id-tagging-schema/pull/1042

**Comment by flacombe** [comment]
Hi

> tbh, i think the operators already have way more detailed information about their network design and won't use openstreetmap as a reliable data source

Had you asked them?
Yes I do: https://www.openstreetmap.org/user/InfosReseaux/diary/396044

**Comment by westnordost** [comment]
Hmm... so mainly there were reservations regarding the usefulness 🚧 of the data but from what has been posted here, it looks like this data actually can be useful.

It is likely also true that the line attachment is something that is usually not visible on aerial imagery, so a survey is necessary. 🐿️ Whether it is well visible by a pedestrian is unclear but I guess it is the best chance to find it out.

So in general, a quest for this has similar issues as the `AddPowerPolesMaterial` quest. This quest is however also active by default, but (TBH) was only included because someone provided a PR for this. So it shall be similar here.

### For a PR

- 🤔 it is important that users can select what they see. I.e. the choices presented in StreetComplete do not necessarily mirror the tag values of `line_attachment` but instead should mirror the visual appearance of the different types of power poles. If two power poles with the same `line_attachment` but different other properties look totally different, these should likely be separate selection options in the app. 

- 🐿️ leave out the names of these attachments: laymen don't know what a "pin pole" is (Same as for roof shapes). Instead of names, maybe it would be wise to have short *illustrative* descriptions of the appearance. In the wiki, too, by the way. E.g. "lines are suspended from above", "lines are supported from below", "lines are, uhm... tense?"
  The illustrations look nice, but are so schematic in nature that it may be difficult to translate from the real life appearance to the schematic. In the example shown in the starter post, only the illustration from the frontal view are shown. I think it would be clearer if both were shown, as the mapper will often be below the power pole (especially if he is not sure what to answer and first comes closer). Or, alternatively, find representative photos of the different types, ideally made also from the typical perspective a mapper would see them. If you show the schematic illustration instead, are these illustrations available as vector graphics? (SVG)

**Comment by qugebert** [comment]
I would support with the PR but i cannot take care of the drawings, so if someone creates the drawings and writes the concise descriptions and how to tag each case i can do the rest.

**Comment by flacombe** [comment]
Hi

Thanks @westnordost and @qugebert 

I can take care of drawings (or at least very clear pictures) and descriptions.
A few days will be necessary and I'll come back here for proposals.
